blog/_posts/btcpay-server-1-2-0.md

```diff
@@ -0,0 +1,101 @@
+
+As the **BTCPay Server 1.2.0** major update includes more than forty items, we won't cover them all in this release post, but as usual the high-time preference crowd can view directly the [full changelog](https://github.com/btcpayserver/btcpayserver/releases).
+
+To update, simply navigate to your **Server Settings > Maintenance** tab and click **Update** or use the command *btcpay-update.sh* in the command line.
+
+
+## **Continuation of our UI updates**
+
+----
+
+Since the beginning of 2021, our Design prodigies at BTCPay Server have been refactoring and updating numerous pages inside the stack to make them less confusing, more comprehensible to a broader userbase and overall, easier to use.
+
+This release includes a number of these updated views, as well as the migration from Bootstrap3 to Bootstrap5. This framework migration enables us to create and implement more modern and user-friendly views.
+
+As examples, the *Crowdfund* and *Point-Of-Sale* creation pages have been updated to make it easier to navigate and more efficiently create your favorite Apps in BTCPay Server.
+
+![BTCPay 1.2.0 Crowdfund view](/images/BTCPay1.2.0Crowdfund.png)
+
+![BTCPay 1.2.0 POS view](/images/BTCPay1.2.0POS.png)
+
+A number of UI improvements and fixes also made it into this release, namely relating to:
+* The server settings view
+* The transaction signing view
+* The payment request view
+* The notifications view
+  
+For the full list, check out the [changelog](https://github.com/btcpayserver/btcpayserver/releases)
+
+## **BTCPay Plugins - Paving the road**
+
+----
+
+In this release, we also introduced the Greenfield API to the internal codebase.
+This allows plugins to gain access to the many internal features they will require in the future.
+
+This addition is paving the road towards having a plugin system in BTCPay that adds infinite functionality and customization on top of the core feature available.
+
+## **Greenfield API**
+
+----
+
+The Greenfield is still being perfected and additions are being made to it in every udpate.
+
+Today, here's what's **new and improved in Greenfield**:
+* Add Delete User
+* Add `CanModifyInvoices` permission
+* Add text search terms to an invoice with `additionalSearchTerms`
+* Add Get store Payment methods
+* Add Generate Store OnChain Wallet
+* Add StoreId to Invoice model
+* Change `enabledOnly` filter to `enabled`
+
+Some API documentation was also corrected to reflect what actually happens when using the endpoints.
+
+## **Translations**
+
+----
+
+Last but not least, BTCPay Server 1.2.0 sees improvements and error corrections in multple translations for the checkout page in:
+* Bulgarian
+* Dutch
+* Finnish
+* Hebrew 
+* Kazath
+* Korean
+* Portuguese
```

Comment:
We should try to credit the contributors who did the updates, should check transifex.

Reply:
I checked transifex and couldn't find anything except wouter for dutch.
The other items come from the changelog additions by @NicolasDorier